homeassistant/components/vlc_telnet/media_player.py

```diff
@@ -238,6 +240,26 @@ def set_volume_level(self, volume):
         self._vlc.set_volume(volume * MAX_VOLUME)
         self._volume = volume
 
+    def volume_up(self):
+        """Service to send VLC the command for volume up."""
+        if self._volume is None:
+            return
+
+        current_volume = self._volume
+        if current_volume < MAX_VOLUME:
+            new_volume = min(current_volume + 0.05, MAX_VOLUME)
```

Comment:
We need to transform the volume from the Home Assistant range (0..1) to the vlc range.

Reply:
Thanks for the speedy review! I might be missing something about the Home Assistant API. From my (manual) testing, it seems to be working as intended. `self.set_volume_level` is handling the mapping.

Reply:
`MAX_VOLUME` volume is the max volume in vlc, not in Home Assistant. Home Assistant has max volume 1.0.

Reply:
Gotcha, good point. Adjusted it to max out at 1.

Reply:
(resolving this since fixed in https://github.com/home-assistant/core/pull/46800/commits/482cc1a3299d0673d1d573d7ccd640ec47a4fa94 -- apologies if I'm getting the etiquette wrong!)